app/views/users/new.erb

```diff
@@ -0,0 +1,9 @@
+
+Fill the spaces to sign up and post messages today
+  <form action="/users/new" method="post">
+  Email:<input type="text" name="email" value=<% @user.email %>>
```

Comment:
This isn't the best way of marking up the form - in particular you're lacking labels which help screen readers understand which form input has which label attached to it:

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/How_to_structure_an_HTML_form#The_<label>_element